distributed/core.py

```diff
@@ -825,12 +825,16 @@ async def handle_stream(self, comm, extra=None):
 
                 if not comm.closed():
                     for msg in msgs:
-                        if msg == "OK":  # from close
+                        if msg == "OK":
                             break
                         op = msg.pop("op")
                         if op:
                             if op == "close-stream":
                                 closed = True
+                                logger.info(
+                                    "Received 'close-stream' from %s; closing.",
+                                    comm.peer_address,
+                                )
```

Comment:
This is a bit verbose, isn't it?

Reply:
`close-stream` should only occur in a few instances, I would like to be able to see when those happen.